pandas/tests/io/test_orc.py

```diff
@@ -1,5 +1,8 @@
-""" test orc compat """
+"""
+test orc compat
+"""
```

Comment:
could you revert unrelated changes (such as this one) please?

Reply:
git reset on this file.